### PR TITLE
cleanup(OWNERS): remove inactive approvers

### PR DESCRIPTION
**Pull request by jasondellaluce**
**What type of PR is this?**

/kind cleanup

/kind documentation

**Any specific area of the project related to this PR?**

/area build

**What this PR does / why we need it**:

For https://github.com/falcosecurity/evolution/issues/157, this PR moves inactive approvers (who had very little or zero contributions over the past 6 months) from `approvers` to `emeritus_approvers` in OWNERS.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by fntlnz — CHANGES_REQUESTED** [review]
Please restore `fntlnz` https://github.com/falcosecurity/falco/issues/2115#issuecomment-1176045086

**Comment by jasondellaluce** [comment]
@fntlnz, given https://github.com/falcosecurity/falco/issues/2122 I understand that you'd like to step down as maintainer of this repository too.

Can you please remove your Change Request here or sign-off your commit on #61 to unblock the DCO there?

Of the two, I would prefer proceeding with this one that puts your name under the `emeritus_approvers` section, let me know what you think.

**Comment by jasondellaluce** [comment]
Closing this in favor of #64.

/close

**Pull request closed without merging**
